packages/notifications-service/README.md

```diff
@@ -1,3 +1,5 @@
+# Notifications Service
+
 # Notifications Microservice
```

Comment:
```suggestion
## Notifications Microservice
```

Reply:
Why `h2`? We're using h1 for all the readme files.

Reply:
Oh wait, that line is duplicated. There should be only 1 heading. I'll remove it.

Reply:
Updated.